frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ImportCluster/ImportCluster.tsx

```diff
@@ -132,6 +135,7 @@ type Action =
   | ({ type: 'setClusterID' } & Pick<State, 'clusterID'>)
   | ({ type: 'setNamespace' } & Pick<State, 'namespace'>)
   | ({ type: 'setCredential' } & Pick<State, 'credential'>)
+  | { type: 'updateCredentials'; payload: { credentialList: ProviderConnection[] } }
```

Comment:
I had mentioned that you do not need to pass in `ocmCredentials` (named `credentialList` here) because the reducer function can already access it. See line 311 for example.